unthink-stack/src/server/resources/version-resource.ts

```diff
@@ -0,0 +1,14 @@
+import * as fs from 'fs';
+import { resource, template, TemplateResponse, RedirectResponse } from 'resource-decorator';
+import { ResourceBase } from './resource-base';
+
+@resource({ basePath: '/unthink/version' })
+export class VersionResource extends ResourceBase {
+
+  @template()
+  async versionPage(): Promise<TemplateResponse | RedirectResponse> {
+    const { name, version } = JSON.parse(fs.readFileSync('./package.json').toString());
```

Comment:
Instead of reading this file in on every request maybe this can be moved out to the config module? The main take away is that we don't want something spamming that version link and the server constantly trying to load that file. 

In the config module we could have something like:

```
export const appName: string = ... from package.json using the method here
export const appVeresion: string = ... from package.json
```
And in your code you then have it available as:

```
config.appVersion
config.appName
```

And then we can also update the other spots that load the package.json just to be consistent like the snippet below. We could then just use `config.appVersion`. 
https://github.com/epandco/unthink-cli/blob/f455667fc6e5289d570d855a993123c29d26969a/unthink-stack/src/server/server.ts#L17-L31

with the config version.

Reply:
Okay yeah that makes sense. I just pushed a commit with the appName and appVersion config. `NunjucksResourceRenderer` now uses the config for `APP_VERSION` and my resource now just passes `config.appName` to the template.

I guess its a little weird that in the template I access appName with `content.APP_NAME` but I access appVersion with just `APP_VERSION` but I don't think its necessary to add appName to `NunjucksResourceRenderer`